.travis.yml

```diff
@@ -65,22 +64,19 @@ matrix:
         - os: linux
           env: ASTROPY_VERSION=development
                EVENT_TYPE='pull_request push cron'
-        - os: linux
-          env: PYTHON_VERSION=2.7 ASTROPY_VERSION=lts
-               PIP_DEPENDENCIES='requests mimeparse requests_mock contextlib2'
```

Comment:
Is this just a copy of the same thing that's run on line 61 here?  I don't understand what the EVENT_TYPE means here in this case though.

Reply:
yes, this is a copy. the other one also runs from cron, it doesn't really have to though as python2 and dependencies is not moving anywhere anymore. The whole travis matrix could use a proper cleanup, adding stages etc, but that's beyond the scope of this PR.